doc/src/devdocs/pkgimg.md

```diff
@@ -33,7 +33,7 @@ Dynamic libraries on macOS need to link against `-lSystem`. On recent macOS vers
 To that effect we link with `-undefined dynamic_lookup`.
 
 ## [Package images optimized for multiple microarchitectures](@id pkgimgs-multi-versioning)
-Similar to [multi-versioning](@ref sysimg-multi-versioning) for system images, package images support multi-versioning. If you are in a heterogenous environment, with a unified cache,
+Similar to [multi-versioning](@ref sysimg-multi-versioning) for system images, package images support multi-versioning. If you are in a heterogeneous environment, with a unified cache,
```

Comment:
It looks like both [`heterogenous`](https://www.merriam-webster.com/dictionary/heterogenous) and [`heterogeneous`](https://www.merriam-webster.com/dictionary/heterogeneous) are valid spellings.

Would you happen to remember which tool (`typos` or `codespell`) flagged `heterogenous` as a misspelling?

Reply:
Pretty sure that heterogeneous is the more common spelling.

Reply:
`codespell`

Reply:
I think it's fine if we standardize on the `heterogeneous` spelling, as long as we're consistent across the codebase.